.github/workflows/verify-api.yaml

```diff
@@ -0,0 +1,53 @@
+name: Verify API
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+jobs:
+  check-api-build:
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/master'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run api in docker container
+        run: |
+          docker buildx build -t epp-biorxiv-xslt:api --target api .
+          docker run -d -p 3000:80 epp-biorxiv-xslt:api
+          sleep 5 # wait for the container to start up
```

Comment:
I'm assuming this works OK, but if it start to become flaky, we should look at better ways to wait for it to come up. (like curl retry mechanism)